src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-size: 12px;
+  font-family: Roboto, serif;
+}
+
+.header {
+  padding: 0 50px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  flex-shrink: 0;
+}
+
+.nav__list {
+  list-style: none;
+
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav__item {
+  position: relative;
+}
+
+.nav__link {
+  color: #000;
+  display: block;
+  font-weight: 500;
```

Comment:
The font-weight of 500 is used here. Ensure that this weight is available in the 'Roboto' font you are importing. If not, it might default to the nearest available weight.